symphony/lib/toolkit/class.datasource.php

```diff
@@ -536,6 +536,38 @@ public static function removeEscapedCommas($string)
         return preg_replace('/(?<!\\\\)\\\\,/', ',', $string);
     }
 
+    private static $digitReplacements = array(
```

Comment:
`static` is not needed, because it's only used in the method `escapeFirstCharIfDigit`. Therefore is `static` also wrong here!

Reply:
static here saves memory. We de create a lot of DataSource object and do not need multiple copies of the data.

Reply:
@nitriques 
> static here saves memory.

Do you have some benchmarks, which illustrates the problem? I found nothing in this repository.
If it is only your feeling, then remove the `static`. 😉

Reply:
> Do you have some benchmarks, which illustrates the problem?

No, but I am pretty confident ;)